build.linux32x86/third-party/Makefile.libgit2

```diff
@@ -56,6 +56,7 @@ install: $(THIRDPARTYLIBDIR)/$(LIBGIT2LIBNAME)
 			cp -d $$each $(THIRDPARTYINSTALLDIR); \
 		fi \
 	done
+	ldd $(THIRDPARTYLIBDIR)/$(LIBGIT2LIBNAME)
```

Comment:
technically, this is not needed anymore either (I put it to see the bindings of libgit2 to ensure it was linking libssh2). Anyway since this can break again I would keep it.

Reply:
Exactly why I didn't comment here. :-)